packages/client/components/SwipeableDashSidebar.tsx

```diff
@@ -98,16 +106,20 @@ const swipe = {
 interface Props {
   children: ReactNode
   isOpen: boolean
+  isRightSidebar?: boolean
   onToggle: () => void
 }
 
 const SwipeableDashSidebar = (props: Props) => {
-  const {children, isOpen, onToggle} = props
+  const {children, isOpen, isRightSidebar = false, onToggle} = props
   const {portal, openPortal} = usePortal({
     allowScroll: true,
     noClose: true
   })
   const [xRef, setX] = useRefState(0)
+  const sidebarWidth: number = isRightSidebar ? DiscussionThreadEnum.WIDTH : NavSidebar.WIDTH
+  const HYSTERESIS_THRESH = HYSTERESIS * sidebarWidth
```

Comment:
nit: i like to reserve all caps for things that are constant at build time (i've probably screwed up & set a bad example somewhere, but that's my method to madness)